index.html

```diff
@@ -3857,6 +3857,78 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section class="informative">
+          <h2>JSON Processing</h2>
+
+          <p>
+While the media types describing conforming documents defined in this
+specification always express JSON-LD, JSON-LD processing is not required to be
+performed, since JSON-LD is JSON. Some scenarios where processing a
+<a>verifiable credential</a> or a <a>verifiable presentation</a> as JSON is
+desirable include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+Before securing, or after verifying content leveraging <a>external proofs</a>
+that require <a href="https://csrc.nist.gov/glossary/term/data_integrity">data
```

Comment:
I think we can simplify this because it's also true for internal proofs -- as the JSON-LD transformations are only needed for some internal proof formats *during* securing or verification, but not before or after:

```suggestion
Before securing or after verifying content
that requires <a href="https://csrc.nist.gov/glossary/term/data_integrity">data
```